app/Models/Post.php

```diff
@@ -23,6 +25,7 @@ class Post extends Model implements HasMedia, Searchable
     use HasTranslations;
     use HasStatuses;
     use InteractsWithMedia;
+    use HasStructuredData;
```

Comment:
Post model implements HasStructuredData too...the idea is to add this Trait to each model we want it to generate tructured data for seo.